views/content/wallet.md

```diff
@@ -0,0 +1,14 @@
+## Ethereum Wallet
+
+The Ethereum Wallet app that you can download allows you to run the full block chain on your own computer. It allows you very easy to be part of the blockchain.
```

Comment:
The Ethereum Wallet is an app that allows you to interact with the Ethereum blockchain on your own computer easily.